core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -218,7 +218,8 @@ object ConsoleConsumer extends Logging {
       .withRequiredArg
       .describedAs("blacklist")
       .ofType(classOf[String])
-    val partitionIdOpt = parser.accepts("partition", "The partition to consume from.")
+    val partitionIdOpt = parser.accepts("partition", "The partition to consume from. Without the " +
```

Comment:
Maybe: "The partition to consume from. Starts from the end of the partition unless `--offset` is specified"